distributed/protocol/tests/test_serialize.py

```diff
@@ -3,10 +3,11 @@
 import pickle
 
 import msgpack
-import numpy as np
 import pytest
 from tlz import identity
 
+np = pytest.importorskip("numpy")
```

Comment:
Most tests in this module don't use NumPy and it'd be good to run them even when NumPy isn't installed. Could you isolate the `importorskip` behavior specifically to tests which need NumPy like you've done in some other test modules?

Reply:
Not possible, because numpy is used at parametrization of tests, i.e. during test collection of the module.